From 65a79fe8ceb52d7ae978f4a2ecc47a1fd34fda2b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20G=C3=B3rny?= <mgorny@gentoo.org>
Date: Sun, 2 Aug 2020 00:11:53 +0200
Subject: [PATCH] Skip test_cov if pytest-cov is not installed

Skip test_cov gracefully if pytest_cov module can't be imported.
This would help us since we're removing py2 support from pytest-cov
but still need it for pytest-timeout.
---
 test_pytest_timeout.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/test_pytest_timeout.py b/test_pytest_timeout.py
index 16302d7..69707b4 100644
--- a/test_pytest_timeout.py
+++ b/test_pytest_timeout.py
@@ -77,6 +77,7 @@ def test_thread(testdir):
 )
 def test_cov(testdir):
     # This test requires pytest-cov
+    pytest.importorskip("pytest_cov")
     testdir.makepyfile(
         """
         import time
-- 
2.28.0

